projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -564,6 +565,33 @@ function wpcom_launchpad_is_keep_building_enabled() {
 	return apply_filters( 'is_launchpad_keep_building_enabled', false );
 }
 
+/**
+ * Filter task visibility for the Keep building task list.
+ *
+ * @param array $task The task array.
+ *
+ * @return array The filtered array of task IDs.
+ */
+function wpcom_launchpad_keep_building_visible_tasks( $task ) {
+	$task_ids = $task['task_ids'];
+
+	if ( ! $task_ids ) {
+		return array();
+	}
+
+	return array_filter(
+		$task_ids,
+		function ( $task_id ) {
+			// Only show design_edited/site_edited if it hasn't been marked as complete.
+			if ( 'design_edited' === $task_id || 'site_edited' === $task_id ) {
+				return is_array( wpcom_is_checklist_task_complete( 'design_edited' ) ) || is_array( wpcom_is_checklist_task_complete( 'site_edited' ) ) ? false : $task_id;
+			}
+
+			return $task_id;
+		}
+	);
```

Comment:
I didn't test it, but it twisted my brain a little. I think that if one of the tasks is complete, they will both be excluded.
```suggestion
	return array_filter(
		$task_ids,
		function ( $task_id ) {
			// Only show design_edited/site_edited if it hasn't been marked as complete.
			if ( in_array( $task_id, [ 'design_edited', 'site_edited' ], true ) ) {
				return ! is_array( wpcom_is_checklist_task_complete($task_id) );
			}

			return true;
		}
	);
```